### DISCUSSION TITLE
Daily goal bar for time goals, word goals, quote goals, etc.

### DISCUSSION THREAD

**Discussion by saieeshb** [monkeytypegame/monkeytype#3632] [Ideas]
I would love a daily goal bar for people (like me) who like to keep a goal of x minutes practiced per day. However, it doesn't need to be just time. It can be like x words typed per day. This pushes some people to get those extra minutes in, even when they don't want to practice.

### Time:

![Untitled-1](https://user-images.githubusercontent.com/66863579/194571051-c1a8060d-19f1-4345-bbfb-a8c8704ac600.png)

### Words:

![words](https://user-images.githubusercontent.com/66863579/194572093-9811fd90-9bef-4709-95b9-8fc7b4afdae7.png)

### You could also represent it in percentages:

![percent time](https://user-images.githubusercontent.com/66863579/194578177-e355c499-1882-4ddf-951b-719b1f2b1fa1.png)

Kind of like the above. It would go invisible when typing. 

You could also make a more _**minimalistic**_ style like this, I don't really like it though because I can't really fit it in to look good. 

![pie](https://user-images.githubusercontent.com/66863579/194577425-398ce8b4-8cae-4590-b2ee-a0c0a615ed21.png)

So this is my idea. Tell me what you guys think. Sorry for the bad concept art, wish I could do better. 

Would love it if this got implemented. I don't really know coding or I would've definitely helped.

Thanks to Jack and the rest of the team once again for this awesome website. <3

**Comment by elburro1887** [comment]
I would also be interested in this, a lot of other typing apps have this feature and it helps the user stay focused and eager to study.

**Comment by pencilled-robin** [comment]
I'd love this feature! I'm currently working on improving my touch typing, and this would be super useful :D

**Comment by Vihershunde** [comment]
I would love this feature!

**Comment by saeedtahmtan** [comment]
I need this feature for daily test count! I will help if you want!
